R/read_jdx.R

```diff
@@ -79,6 +74,10 @@ read_jdx <- function(file = stop("filename is needed"), SOFC = TRUE, debug = 0)
       packageDescription("hySpc.read.jdx")$BugReports
     )
   }
+
+  # Output
+  .spc_io_postprocess_optional(spc, filename = file)
+  # return(list(metadata = list_jdx[[2]], hyperSpec = spc))
```

Comment:
please remove this commented-out line of code

Reply:
I just moved the two rows outside the if-else statement. I'm not sure why the commented code was left. But, in general, I agree that there should not be any commented code left in the packages.

Reply:
Done.